docs/book/src/testing/unit-testing.md

```diff
@@ -92,3 +92,37 @@ fn test_fail() {
 ```
 
 > **Note:** When running `forc test`, your contract will be built twice: first *without* unit tests in order to determine the contract's ID, then a second time *with* unit tests with the `CONTRACT_ID` provided to their namespace. This `CONTRACT_ID` can be used with the `abi` cast to enable contract calls within unit tests.
+
+Contracts can call other contract's functions if they are declared unedr [`contract-dependencies`](../forc/manifest_reference.md#the-contract-dependencies-section) table. An example for such calls can be seen below:
+
+```sway
```

Comment:
I'd love for these examples to be moved into the `examples` directory if possible.